docs/about.md

```diff
@@ -0,0 +1,21 @@
+# About KLoadGen
+
+KLoadGen is a plugin for JMeter which acts as a load generator designed to send Kafka messages and that supports both AVRON and JSON structures. 
+
+It can also connect to the Schema Registry server, where it retrieves the subjects to send.
+
+## Benefits
+
+- Test your performance and the business logic. You can easily test both the performance of your system and the business logic behind your data.
+- Define schemas for your test. You can specify a file with the schema or configure the connection to the Schema Registry server on the go.
+- Generate synthetic data. You can set schema constraints to generate random data that matches your structure.
+- Generate key values. You can also define keys that match your structure while improving your cluster performance.
+- Dependency-free. You don't need external libraries and there are no other dependencies; the plugin automatically embeds the latest supported version of Kafka libraries.
+- AVRO and JSON. This plugin supports both AVRO and JSON schemas.
+- Arrays and maps. KLoadgen generates messages both from basic data types and from complex structures, such as arrays or maps.
+
+## Product roadmap
```

Comment:
I don't know if you are describing this section right. In the roadmap's section, you should link to where you explain what is the way forward for this product, what the plans are, and what is currently being worked on. 

You are describing the contributing model in this section. Something that you are already telling on the Additional Information page. Check if KLoadGen has a roadmap, what materials are available, or how they want to explain this section.

Reply:
You are right. I am asking the developers to provide the content for this section.

Reply:
Text already changed, could you please recheck, @MattCodesido?